app/models/nsm_claim.rb

```diff
@@ -3,5 +3,13 @@ class NsmClaim < ApplicationRecord
   has_one :assigned_counsel_claim, dependent: :destroy
   has_one :submission, dependent: :destroy
 
-  validates :laa_reference, presence: true
+  validates :laa_reference,
+            presence: true
+
+  validates :ufn, ufn: true, on: :update
+  validates :office_code, format: { with: /\A\d[a-zA-Z0-9]*[a-zA-Z]\z/,
+                                    message: I18n.t("errors.assigned_counsel_claim.counsel_office_code"),
```

Comment:
could use a custom validator for this as it's reused in assigned_counsel_claim (twice)